pkg/api/v1/conversion.go

```diff
@@ -24,12 +24,22 @@ import (
 	"k8s.io/kubernetes/pkg/conversion"
 )
 
+const (
+	// Annotation key used to identify mirror pods.
+	mirrorAnnotationKey = "kubernetes.io/config.mirror"
```

Comment:
I realize this is too late now, but, for the future, this should have been something like `kubelet.kubernetes.io/mirror-config`. Prefix before the slash should be at least kubelet-specific, and suffix after the slash should use dashes rather than dots.